contracts/LifToken.sol

```diff
@@ -45,14 +45,18 @@ contract LifToken is MintableToken, Pausable {
      @param data ABI-encoded contract call. For example generated using web3's
      getData method
    */
-  function approveData(address spender, uint value, bytes data) {
+  function approveData(address spender, uint value, bytes data) whenNotPaused returns (bool) {
```

Comment:
can you please add the `@return true ...` to the doxygen comment?

Reply:
Yes! What do you think of : 
`@return true if the call function was executed successfully`
On the three transfer data functions?

Reply:
what about `@return true if the call with data did not throw`?